Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14-alpine
+
+WORKDIR /excalidraw-room
+
+COPY package.json yarn.lock ./
+COPY scripts ./scripts
+RUN SKIP_YARN_POSTINSTALL=true yarn
```

Comment:
What's the purposes of `YARN_POSTINSTALL` var?

Reply:
See https://github.com/excalidraw/excalidraw-room/pull/71#discussion_r454314381 but essentially on `postinstall` this repo would previously run `yarn build`. However, in order to leverage layer caching (https://github.com/excalidraw/excalidraw-room/pull/71#discussion_r454320273), I had to separate `yarn install` and `yarn build` commands and essentially skip the `postinstall` script.

Reply:
Ah I see, that's because we deploy on Heroku and we need to have everything built by the time Heroku starts the node.

I just see it as too much complication for such a simple process. I wonder if we could find another way to deploy to Heroku without having to have this `postinstall` script.

Reply:
What if it were moved to `start`?

```js
{
  "start": "yarn build && node ./dist/index.js"
}
```

Reply:
I'm not sure how Heroku will behave if we do that. I would be up for it definitely. Will make it much simpler.

I noticed that there is #13 which is related. We could have a GH action to push the docker image and a GH action to push to Heroku.

Reply:
> I'm not sure how Heroku will behave if we do that. I would be up for it definitely. Will make it much simpler.

Maybe leave it for now and potentially revisit in a follow-up? What I have now shouldn't affect the existing flow. Also it's hard for me to test without access to Heroku.

Reply:
No worries, I'm here to help. :) I'm looking into how we can deploy the same Dockerfile to Heroku. It may be slighter more complicated but will clear things up. If we bump into issues, we will just go as is.

Reply:
https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process

This seems to suggest that the presence of `build` script should be enough for Heroku?

Reply:
Ah yes, if that works, that's great. I can try it on my Heroku sandbox and let you know or feel free to try it your self.

Reply:
I don't have a Heroku account so it'd be great if you can. But if it takes you a while, I can try to find some time and get set up :)

Reply:
I will, dont worry.

Reply:
<details>
<summary>
I deleted postinstall and it worked:
</summary>

```
$ git push heroku master

Enumerating objects: 145, done.
Counting objects: 100% (145/145), done.
Delta compression using up to 8 threads
Compressing objects: 100% (55/55), done.
Writing objects: 100% (145/145), 113.07 KiB | 37.69 MiB/s, done.
Total 145 (delta 72), reused 142 (delta 71)
remote: Compressing source files... done.
remote: Building source:
remote: 
remote: -----> Node.js app detected
remote:        
remote: -----> Creating runtime environment
remote:        
remote:        NPM_CONFIG_LOGLEVEL=error
remote:        USE_YARN_CACHE=true
remote:        NODE_ENV=production
remote:        NODE_MODULES_CACHE=true
remote:        NODE_VERBOSE=false
remote:        
remote: -----> Installing binaries
remote:        engines.node (package.json):  unspecified
remote:        engines.npm (package.json):   unspecified (use default)
remote:        engines.yarn (package.json):  unspecified (use default)
remote:        
remote:        Resolving node version 12.x...
remote:        Downloading and installing node 12.18.2...
remote:        Using default npm version: 6.14.5
remote:        Resolving yarn version 1.x...
remote:        Downloading and installing yarn (1.22.4)...
remote:        Installed yarn 1.22.4
remote:        
remote: -----> Installing dependencies
remote:        Installing node modules (yarn.lock)
remote:        yarn install v1.22.4
remote:        [1/4] Resolving packages...
remote:        [2/4] Fetching packages...
remote:        [3/4] Linking dependencies...
remote:        [4/4] Building fresh packages...
remote:        $ npm run build
remote:        
remote:        > excalidraw-room@1.0.0 build /tmp/build_3af80b9d
remote:        > tsc
remote:        
remote:        Done in 10.76s.
remote:        
remote: -----> Build
remote:        Running build (yarn)
remote:        yarn run v1.22.4
remote:        $ tsc
remote:        Done in 2.58s.
remote:        
remote: -----> Pruning devDependencies
remote:        yarn install v1.22.4
remote:        [1/4] Resolving packages...
remote:        [2/4] Fetching packages...
remote:        [3/4] Linking dependencies...
remote:        [4/4] Building fresh packages...
remote:        warning Ignored scripts due to flag.
remote:        Done in 1.34s.
remote:        
remote: -----> Caching build
remote:        - yarn cache
remote:        
remote: -----> Build succeeded!
remote: -----> Discovering process types
remote:        Procfile declares types     -> (none)
remote:        Default types for buildpack -> web
remote: 
remote: -----> Compressing...
remote:        Done: 29.4M
remote: -----> Launching...
remote:        Released v3
remote:        https://afternoon-peak-26062.herokuapp.com/ deployed to Heroku
remote: 
remote: Verifying deploy... done.
To https://git.heroku.com/afternoon-peak-26062.git
 * [new branch]      master -> master

```

</details>

Going to re-push this.